code/API_definitions/qod-api.yaml

```diff
@@ -328,6 +477,166 @@ components:
           - 5070
       description: |
         Ports may be specified as a list of ranges or single ports.
+    QoSProfileId:
+      type: string
+      example: 123e4567-e89b-12d3-a456-426614174000
+      description: |
+        Unique ID of the QoS Profile to be used for this session.
+      minLength: 36
+      maxLength: 36
+      format: string
+    ExtendedQosProfile:
+      description: Data type with attributes of a QosProfile
+      type: object
+      properties:
+        id:
+          $ref: "#/components/schemas/QoSProfileId"
+        name:
+          type: string
+          example: QCI_1_voice
+          description: Descriptive name for QoS Profile
+          minLength: 3
+          maxLength: 128
+          format: string
+        status:
+          $ref: "#/components/schemas/QosProfileStatusEnum"
+          description: The status of the profile.  See QosProfileStatusEnum for description.
+        targetMinUpstreamRate:
+          $ref: "#/components/schemas/Rate"
+          description: The target minimum data rate.  Set to zero when no target minimum is defined in this profile.
+        maxUpstreamRate:
+          $ref: "#/components/schemas/Rate"
+          description: |
+            The maximum best effort data
+            Equal to targetMinimumUpstreamRate when max is the as the target. 
+        maxUpstreamBurstRate:
+          $ref: "#/components/schemas/Rate"
+          description: |
+            The burst rate for best effort data
+        targetMinDownstreamRate:
+          $ref: "#/components/schemas/Rate"
+          description: The target minimum data rate.  Set to zero when no target minimum is defined in this profile.
+        maxDownstreamRate:
+          $ref: "#/components/schemas/Rate"
+          description: |
+            The maximum best effort rate
+            Equal to targetMinimumUpstreamRate when max is the as the target. 
+        maxDownstreamBurstRate:
+          $ref: "#/components/schemas/Rate"
+          description: |
+            The maximum best effort burst rate
+            Equal to targetMinimumUpstreamRate when max is the as the target. 
+        minDuration:
+          $ref: "#/components/schemas/Duration"
+          description: The shortest time period that is supported this profile can be deployed.
+        maxDuration:
+          $ref: "#/components/schemas/Duration"
+          description: The maximum length of time that this profile can be deployed.
+        priority:
+          type: number
+          example: 5.0
+          description: |
+            Priority for this session
```

Comment:
`description: The priority that will be given to the session, where a lower value indicates a higher priority`